dts/bindings/i2c/i2c-controller.yaml

```diff
@@ -17,3 +17,7 @@ properties:
   clock-frequency:
     type: int
     description: Initial clock frequency in Hz
+  startup-delay-us:
```

Comment:
this seems like DT being used for software configuration, is this based on a datasheet? if so, is it based on the lpi2c controller characteristics or some i2c device characteristics? if it is a software configuration or based on the device characteristics maybe it should not be in the lpi2c binding.

Reply:
@decsny No this is not based on the datasheet. This is based on the behavior of the board. What do you mean by software configuration? A lot of things in the DT configure the software to behave in some way as the hardware dictates.

Anyhow as I said, I wanted to put this up for discussion. What would be the right way in your opinion?

Reply:
I don't like this, it seems like a bit of a specific fix for a specific board for a specific peripheral IP.

I get the desire to maybe delay after initializing a device though, so perhaps making it a general struct device aware DT binding and struct device delay call. E.g. post-init-delay would also be clearer than startup delay to me.